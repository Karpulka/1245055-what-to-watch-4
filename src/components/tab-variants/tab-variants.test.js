import React from "react";
import renderer from "react-test-renderer";
import TabVariants from "./tab-variants";

const tab = {
  title: `Overview`,
  value: {
    description: `This is Description. TCHK.`,
    rating: 8.5,
    voiceCount: 214,
    director: `And he's Director`,
    actorList: [`Actor`, `Actor`, `Actor`]
  }
};

it(`Render TabVariants No Active Tab`, () => {
  const tree = renderer
    .create(<TabVariants tab={tab} active={`Details`} onTabClick={() => {
    }}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render TabVariants Active Tab`, () => {
  const tree = renderer
    .create(<TabVariants tab={tab} active={`Overview`} onTabClick={() => {
    }}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
