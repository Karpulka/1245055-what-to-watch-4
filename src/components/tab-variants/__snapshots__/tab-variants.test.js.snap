// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render TabVariants Active Tab 1`] = `
<li
  className="movie-nav__item movie-nav__item--active"
>
  <a
    className="movie-nav__link"
    href="/"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;

exports[`Render TabVariants No Active Tab 1`] = `
<li
  className="movie-nav__item"
>
  <a
    className="movie-nav__link"
    href="/"
    onClick={[Function]}
  >
    Overview
  </a>
</li>
`;
